Presentation.Web/app/services/userServices.ts

```diff
@@ -369,10 +381,17 @@
 
         };
 
-        chooseOrganization = () => {
+        chooseOrganization = (orderBy?: string, orderByAsc?: boolean) => {
```

Comment:
do you need these parameters on this method?
it's called by another method in the same class and if we agree that we always want ordering by name asc in the "chose org" use case, then just use those parameters when calling getOrganizations